Portfolio/index.css

```diff
@@ -1,174 +1,126 @@
-console.log('Hello World');

```

Comment:
The * selector on line 1 should be removed, as it targets all elements and is unnecessary.